<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageCurrency_Details</class>
 <widget class="QWizardPage" name="PageCurrency_Details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Contract name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditContractName">
     <property name="placeholderText">
      <string>e.g. MtGox Dollar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Primary Unit Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditPrimaryUnit">
     <property name="placeholderText">
      <string>e.g. dollars</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Symbol:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEditFractionalUnit">
     <property name="placeholderText">
      <string>e.g. cents</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEditSymbol">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>e.g. $</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>TLA:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEditTLA">
     <property name="placeholderText">
      <string>e.g. USD</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Fractional Unit Name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Decimal spaces:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEditDecimalSpaces">
     <property name="placeholderText">
      <string>e.g. 2 (as shown: $1.00)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonDefaults">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Use Defaults</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
